src/groovy/org/transmartproject/export/GENExporter.groovy

```diff
@@ -89,13 +92,13 @@ class GENExporter implements HighDimTabularResultExporter {
                 out << ' '
                 out << row.a2
                 
-                for (/*SnpLzCell*/ Object cell: row) { 
+                for (double[] cell: row) {
                     out << ' '
-                    out << cell.probabilityA1A1
+                    out << cell[0] // probabilityA1A1
                     out << ' '
-                    out << cell.probabilityA1A2
+                    out << cell[1] // probabilityA1A2
                     out << ' '
-                    out << cell.probabilityA2A2
+                    out << cell[2] // probabilityA2A2
```

Comment:
Sure, but changing rest-api to support this was not a two liner. Currently a value from a non-MultiValueProjection is assumed to be a scalar, and the contract of a MultiValueProjection is that whatever it returns has the properties and types given by the projection's dataProperties map. AFAIK array elements cannot be accessed as a property (and we want to use more descriptive names than '0', '1', '2' in the rest output), so this requires changing core-api.